### ISSUE TITLE
Please document the incredibly useful bufio.ts

### ISSUE THREAD

**Issue by suchislife801** [denoland/deno#7267]
I simply cannot find ANY documentation on the following extremely useful imports:

import { BufReader, BufWriter } from 'https://deno.land/std@0.67.0/io/bufio.ts';
import { TextProtoReader }      from 'https://deno.land/std@0.67.0/textproto/mod.ts';

Can these please be documented. I'm a huge fan of Deno but right now I am literally copying & pasting code knowing that's going to break and hours later it eventually works but I have no idea how. It's like coding in the dark.

**Comment by lucacasonato** [comment]
@suchislife801 I agree a readme would be nice. Does https://doc.deno.land/https/deno.land/std@0.67.0/io/bufio.ts and https://doc.deno.land/https/deno.land/std@0.67.0/textproto/mod.ts help?

**Comment by suchislife801** [comment]
Not at all.
